### PR TITLE
Intl versions

### PR DESCRIPTION
**Pull request by altotunchitoo**
Resolved intl pub version.

### CODE REVIEW

**Review by MominRaza — APPROVED** [review]
LGTM
It fixes this error

Because timeago >=3.3.0 depends on intl ^0.17.0 and every version of flutter_localizations from sdk depends on intl 0.18.0, timeago >=3.3.0 is incompatible with flutter_localizations from sdk.
So, because yourapp depends on both flutter_localizations from sdk and timeago ^3.3.0, version solving failed.
exit code 1

**Comment by simplenotezy** [comment]
LGTM! Let's get this merged

**Comment by simplenotezy** [comment]
@andresaraujo I tried manually pulling your project in, but that fails:

```yaml
  timeago: #^3.3.0
    git:
      url: https://github.com/altotunchitoo/timeago.dart.git
      ref: 4cf0bd19b02738a7efd0112092a9872d9286669d # commit hash
      path: ./packages/timeago/
```

Error:
```
❯ flutter pub get
Resolving dependencies...
Because every version of timeago from git depends on intl ^0.17.0 and app depends on intl ^0.18.0, timeago from git is
  forbidden.
So, because app depends on timeago from git, version solving failed.

File: packages/timeago_flutter/pubspec.yaml [hunk 1]
```diff
@@ -7,12 +7,15 @@ environment:
   sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  timeago: ^3.3.0
```
Comment (simplenotezy) [inline packages/timeago_flutter/pubspec.yaml:10]: I would keep this change, and not add the manual dependencies override in your PR

**Review by simplenotezy — CHANGES_REQUESTED** [review]

File: packages/timeago_flutter/pubspec.yaml (hunk 1 above)
Comment (altotunchitoo) [reply packages/timeago_flutter/pubspec.yaml]: Oh, okay...

**Comment by andresaraujo** [comment]
v3.4.0 Has been released and should fix this. I will close this PR thanks. Feel free to ping me if still causes issues

**Pull request closed without merging**